src/utils/wrapper.ts

```diff
@@ -0,0 +1,89 @@
+import * as grpc from 'grpc';
+import * as apm from 'elastic-apm-node';
+import * as _ from 'lodash';
+import { Severity, log } from "./logger";
+import { statusToString, validateGrpcError } from './errors/grpc.status';
+import { ApplicationError } from './errors/errors';
+
+/**
+ * extracts the wanted information for the logger from the response.
+ * for example: extracts only the file ids from the returned array of files.
+ * @param res - the result of the method called in file.grpc
+ */
+function extractResLog(res: any): object {
+    if (!res) return {};
+
+    const logInfo = _.cloneDeep(res);
+    if (res.files) {
```

Comment:
`res.files`?

Reply:
This file is was copied from file-service.
I forgot to change it.